packages/mariadb-connection-module/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mariadb-connection-module",
+  "version": "1.0.0",
```

Comment:
Usually products start in `0.1.0` when they are in development. Using a `1.0.0` means that we have a finished first version.